sheepdog/transactions/upload/entity.py

```diff
@@ -21,6 +21,19 @@
 from sheepdog.transactions.entity_base import EntityBase, EntityErrors
 from sheepdog.utils import get_suggestion
 
+# TODO: This should probably go into the dictionary and be
```

Comment:
why don't we put it in the dictionary?

Reply:
Because it's a review to do so, and because a dictionary update has to go out before a release...if we miss it, we don't have the ability to make the change. Fun.